CHANGELOG.next.md

```diff
@@ -10,6 +10,8 @@ Thanks, you're awesome :-) -->
 
 ### Schema Changes
 
+* Align input options for --include and --subset arguments #1519
```

Comment:
This is more of a tooling addition than a schema change, right?

Reply:
Indeed, thanks for spotting @kgeller